mlrun/package/packagers/python_standard_library_packagers.py

```diff
@@ -35,6 +35,29 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 
+class NonePackager(DefaultPackager):
+    """
+    `None` packager.
+    """
+
+    # TODO: From python 3.10 the `PACKABLE_OBJECT_TYPE` should be changed to `types.NoneType`
+    PACKABLE_OBJECT_TYPE = type(None)
+    DEFAULT_PACKING_ARTIFACT_TYPE = ArtifactType.RESULT
+
+    # TODO: `None` as pickle will be available from Python 3.10, so this method can be removed once we move to 3.10.
+    @classmethod
+    def get_supported_artifact_types(cls) -> List[str]:
+        """
+        Get all the supported artifact types on this packager. It will be the same as `DefaultPackager` but without the
+        'object' artifact type support.
```

Comment:
Not sure i understand why go this way and just use the "result" type instead of using the full list and remove the "object" type.

Reply:
If in the future more artifact types would be included in the `DefaultArtifact` it will be added without the `"object"` automatically. That's the one and only reason :)